File: src/main/scala-graphql/io/github/graphql/scala/model/SecurityAdvisoriesQueryResponse.scala
package io.github.graphql.scala.model

import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLResult
import java.util.{ Map => JMap }

@javax.annotation.Generated(
    value = Array("com.kobylynskyi.graphql.codegen.GraphQLCodegen"),
    date = "2021-03-07T19:49:37+0800"
)
class SecurityAdvisoriesQueryResponse extends GraphQLResult[JMap[String, SecurityAdvisoryConnectionTO]] {

    def securityAdvisories(): SecurityAdvisoryConnectionTO = {
        val data: JMap[String, SecurityAdvisoryConnectionTO] = getData
        if (data != null) data.get(SecurityAdvisoriesQueryResponse.OPERATION_NAME) else null
    }

}

object SecurityAdvisoriesQueryResponse {

    private final val OPERATION_NAME: String = "securityAdvisories"

}

File: src/main/java-graphql/io/github/graphql/model/SecurityAdvisoriesQueryResponse.java
package io.github.graphql.model;

import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLResult;
import java.util.Map;

@javax.annotation.Generated(
    value = "com.kobylynskyi.graphql.codegen.GraphQLCodegen",
    date = "2021-03-06T19:47:48+0800"
)
public class SecurityAdvisoriesQueryResponse extends GraphQLResult<Map<String, SecurityAdvisoryConnectionTO>> {

    private static final String OPERATION_NAME = "securityAdvisories";

    public SecurityAdvisoriesQueryResponse() {
    }

    public SecurityAdvisoryConnectionTO securityAdvisories() {
        Map<String, SecurityAdvisoryConnectionTO> data = getData();
        return data != null ? data.get(OPERATION_NAME) : null;
    }

}
